modules/store.js

```diff
@@ -0,0 +1,30 @@
+// STORE CLASS: Handles storage
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(index) {
+    const books = Store.getBooks();
+
+    books.forEach((book, i) => {
+      if (i === index) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] It will be great to convert these functions into arrow functions. 

**hint**

```
static getBooks = () => {
   your code
}

```